pkg/sdk/policy_references.go

```diff
@@ -0,0 +1,112 @@
+package sdk
+
+import (
+	"context"
+	"database/sql"
+)
+
+type PolicyReferences interface {
+	GetForEntity(ctx context.Context, request *GetForEntityPolicyReferenceRequest) ([]PolicyReference, error)
+}
+
+type getForEntityPolicyReferenceOptions struct {
+	select_       bool           `ddl:"static" sql:"SELECT"`
+	asterisk      bool           `ddl:"static" sql:"*"`
+	from          bool           `ddl:"static" sql:"FROM"`
+	tableFunction *tableFunction `ddl:"keyword"`
+}
+
+type tableFunction struct {
+	table                   *bool                    `ddl:"keyword" sql:"TABLE"`
+	policyReferenceFunction *policyReferenceFunction `ddl:"list,parentheses,no_comma"`
+}
+
+type policyReferenceFunction struct {
+	functionFullyQualifiedName *bool                             `ddl:"keyword" sql:"SNOWFLAKE.INFORMATION_SCHEMA.POLICY_REFERENCES"`
+	arguments                  *policyReferenceFunctionArguments `ddl:"list,parentheses"`
+}
+type policyReferenceFunctionArguments struct {
+	refEntityName   *string `ddl:"parameter,single_quotes,arrow_equals" sql:"ref_entity_name"`
```

Comment:
I also wonder if refEntityName should not be ObjectIdentifier and if we should not run fully qualified name on it, due to this part of the docs:

```
REF_ENTITY_NAME => 'string'
The name of the object, such as the table name, view name, external table name, or username, on which the policy is set.

The entire object name must be enclosed in single quotes.

If the object name is case-sensitive or includes any special characters or spaces, double quotes are required to process the case/characters. The double quotes must be enclosed within the single quotes (i.e. '"<table_name>"').
```

It was also implemented like that for test usage (check `getRowAccessPolicyFor` func).